index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="Author" content="Shubh M Das">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+
+<body>
+   <header>
+        <div class="container">
+            <div id="branding">
+                <h3>My Logo</h3>
+            </div>
+            <nav>
+                <ul>
+                 <li><a href="#">Portfolio</a></li> 
+                 <li><a href="#">About</a></li>
+                 <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+
+        </div>
+   </header>
+
+   <section class="showcase">
```

Comment:
- [ ] Please, consider that your headline section has the margins (top, bottom, left, right) defined in the design. You should also consider the content alignment (your version is aligned center, not like in Figma design)

> Figma design
<img width="321" alt="Screen Shot 2022-02-10 at 15 23 28" src="https://user-images.githubusercontent.com/49064106/153397367-b8d81ae5-8e00-4978-8e92-3a4a58c1a405.png">